docs/src/pages/theming.mdx

```diff
@@ -0,0 +1,762 @@
+import { Callout } from "nextra-theme-docs";
+
+import { UploadButton, UploadDropzone } from "@uploadthing/react";
+
+# Theming
```

Comment:
Only thought here - it'd be nice to have a minimal example up top and a bit more context. Not your responsibility, will put it on my TODO